tracker/tracker/src/main/modules/mouse.ts

```diff
@@ -120,48 +120,58 @@ export default function (app: App): void {
     }
   }
 
-  const selectorMap: { [id: number]: string } = {}
-  function getSelector(id: number, target: Element): string {
-    return (selectorMap[id] = selectorMap[id] || _getSelector(target))
+  const patchDocument = (document: Document) => {
+    const selectorMap: { [id: number]: string } = {}
+    function getSelector(id: number, target: Element): string {
+      return (selectorMap[id] = selectorMap[id] || _getSelector(target, document))
+    }
+
+    app.attachEventListener(document.documentElement, 'mouseover', (e: MouseEvent): void => {
+      const target = getTarget(e.target, document)
+      if (target !== mouseTarget) {
+        mouseTarget = target
+        mouseTargetTime = performance.now()
+      }
+    })
+    app.attachEventListener(
```

Comment:
might be a good idea to add new const `const TRACKING_DELAY = 10` so we can configure how often this event is fired (to prevent tracker from eating cpu)